lib/widgets/customTextField.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+class CustomTextField extends StatelessWidget {
```

Comment:
[Also, the name of file should follow dart name convention.
https://dart.dev/effective-dart/style](https://github.com/GDSC-IAU/Task-1T2/pull/8/files#r1532403054)